Add guideline for secret scanning (#95)
equinor/appsec: docs/guidelines/index.md

---
layout: page
title: Appsec guidelines
---

## Equinor Appsec guidelines

> This section contains guidelines relevant for Equinor AppSec.

Reviewer: ```suggestion
This section contains guidelines relevant anyone writing code in Equinor.
```
